### PR TITLE
VP-59: Allow to set foreignSource for requisitions

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/VP-59

### CODE REVIEW

File: docs/modules/configuration/pages/requisitions.adoc [hunk 1]
```diff
@@ -25,3 +25,10 @@ The following example demonstrates the creation of a customer requisition by usi
 ```
 opennms:import-requisition velocloud-customer alias=aPartnerAlias,enterpriseId=123
 ```
+
+You can specify an additional parameter `foreignSource` to set the foreign source and requisition name.
```
Comment (Bonrob2) [inline docs/modules/configuration/pages/requisitions.adoc:29]:
```suggestion
You can specify an additional parameter, `foreignSource`, to set the foreign source and requisition name.
```

File: docs/modules/configuration/pages/requisitions.adoc [hunk 2]
```diff
@@ -25,3 +25,10 @@ The following example demonstrates the creation of a customer requisition by usi
...
+The following example will create a customer requisition named `My-Customer-Requisition`.
```
Comment (Bonrob2) [inline docs/modules/configuration/pages/requisitions.adoc:30]:
```suggestion
The following example creates a customer requisition named `My-Customer-Requisition`.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor changes to the docs. Thank you!

File: docs/modules/configuration/pages/requisitions.adoc (hunk 1 above)
Comment (christianpape) [reply docs/modules/configuration/pages/requisitions.adoc]: done.

**Review by Bonrob2 — APPROVED** [review]
docs LGTM! Thank you!

**Review by aramosvizcarra — APPROVED** [review]
LGTM

File: plugin/src/main/java/org/opennms/velocloud/requisition/PartnerRequisitionProvider.java [hunk 3]
```diff
@@ -48,6 +48,8 @@
 
 public class PartnerRequisitionProvider extends AbstractRequisitionProvider<PartnerRequisitionProvider.Request> {
 
+    public static final String PARAMETER_FOREIGN_SOURCE = "foreignSource";
```
Comment (fooker) [inline plugin/src/main/java/org/opennms/velocloud/requisition/PartnerRequisitionProvider.java:51]: Push it to the parent class?

**Review by fooker — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/org/opennms/velocloud/requisition/PartnerRequisitionProvider.java (hunk 3 above)
Comment (christianpape) [reply plugin/src/main/java/org/opennms/velocloud/requisition/PartnerRequisitionProvider.java]: done.

**Review by fooker — APPROVED** [review]

**Pull request merged**